inc/customize/customize-design.php

```diff
@@ -106,6 +106,53 @@ function lightning_customize_register_design( $wp_customize ) {
 			)
 		);
 	} // if ( apply_filters( 'lightning_show_default_keycolor_customizer', true ) ) {
+	
+	// defualt eye cache hedding.
```

Comment:
```suggestion
	// Defualt thumbnail hedding.
```

|・ｗ・）.oO（ アイキャッチは eye catch やな... ）